.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
Before asking questions below I'd ask about context in which dlt is used: ie. we could ask
- I'm already dlt user
- I'd consider to use dlt but I'm lacking a feature
- I run dlt in production
- I run it for fun
etc.